app/Http/Controllers/Api/KeywordController.php

```diff
@@ -8,16 +8,33 @@
 use App\Video;
 
 
+
 class KeywordController extends Controller
 {
-    public function getTheVideos() {
-        $keyword_id = 1;
-        $keyword = Keyword::findOrFail($keyword_id);
-        $videos = $keyword->videos()->get();
-        return [
-            $videos,
-            $keyword
-        ];
+    public function getTheKeywords() {
+        // $keyword_id = 1;
```

Comment:
It's usually a bad idea to commit commented out code. Unless you're going to uncomment it very soon.

You can always get back old code using git instead. This is not, critical, can leave for now.